frontend/public/components/utils/resource-log.tsx

```diff
@@ -412,6 +524,10 @@ export const ResourceLog: React.FC<ResourceLogProps> = ({
           namespaceUID={namespaceUID}
           toggleWrapLines={setWrapLines}
           isWrapLines={wrapLines}
+          hasPreviousLog={hasPreviousLogs}
+          changeLogType={setLogType}
+          logType={logType}
+          isContainer={resource.kind === 'Pod'}
```

Comment:
Maybe we should call this something like `showLogTypeSelect` if that's how it's used? Builds run pods which also have containers, so this might be confusing otherwise.

Reply:
Very good point. I agree that name, wasn't quite right.  Changed it to `showLogTypeSelect`